### PR TITLE
Fix exit code delegation.

### PR DESCRIPTION
**Pull request by dereuromark**
Properly delegate exit code

Resolves https://github.com/cakephp/migrations/issues/426

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -80,6 +77,3 @@ script:
 
 after_success:
   - if [[ $CODECOVERAGE == 1 ]]; then bash <(curl -s https://codecov.io/bash); fi
-
-notifications:
-  email: false
```
Comment (ADmad) [inline .travis.yml]: This seems to be mistakenly removed.

Comment (dereuromark) [reply .travis.yml]: Nope, that is void stuff that is not needed :) Trust me.

Comment (ADmad) [reply .travis.yml]: I'll take your word for it, just want to start getting notification emails for every Travis build :)

Comment (dereuromark) [reply .travis.yml]: I feel you : )

Comment (dereuromark) [reply .travis.yml]: I will double check to make sure.

**Pull request merged**